### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by bhaweshchaudhary** [vercel/community#4107] [Account Issue] [answered]
### Question

I've been trying to login to my vercel account, it always ask me to put my phone number, but when i do that, i get this message "This phone number is already associated with a Vercel account."
And nothing happens after that, i can't login due to this.
<img width="540" alt="Screenshot 2023-09-16 at 11 43 11 AM" src="https://github.com/vercel/community/assets/45531771/21810b86-330c-403e-9ce5-c07344ac1747">

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at [registration@vercel.com](mailto:registration@vercel.com)
